app/models/entrant.rb

```diff
@@ -0,0 +1,33 @@
+class Entrant < ApplicationRecord
+  belongs_to :raceable, polymorphic: true, touch: true
+  belongs_to :user
+
+  def ready?
+    readied_at.present?
+  end
+
+  def finished?
+    finished_at.present?
+  end
+
+  def forfeited?
+    forfeited_at.present?
+  end
+
+  def done?
+    finished? || forfeited?
+  end
+
+  def place
+    return '-' unless finished?
```

Comment:
Daydreaming about the future when we can fill this in even when not `finished?` due to having splits in races 😍😍